nixos/doc/manual/release-notes/rl-2605.section.md

```diff
@@ -32,6 +32,8 @@
 
 - [Komodo Periphery](https://github.com/moghtech/komodo), a multi-server Docker and Git deployment agent by Komodo. Available as [services.komodo-periphery](#opt-services.komodo-periphery.enable).
 
+- [sqlit-tui](https://github.com/Maxteabag/sqlit), the lazygit of SQL databases - a lightweight TUI for SQL Server, PostgreSQL, MySQL, SQLite, and more.
+
```

Comment:
```suggestion
```
Package additions do not need to be on the changelog. Only new NixOS modules need to be.